unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/FriendsController/IFriendsController.cs

```diff
@@ -45,8 +45,8 @@ public interface IFriendsController
         UniTask<SocialFriendRequest> RejectFriendshipAsync(string friendRequestId);
         bool IsFriend(string userId);
         void RemoveFriend(string friendId);
-        void GetFriends(int limit, int skip);
-        void GetFriends(string usernameOrId, int limit);
+        UniTask<string[]> GetFriendsAsync(int limit, int skip, CancellationToken ct);
```

Comment:
its a common practice to set the cancellation token with a default value: `CancellationToken cancellationToken = default`

Reply:
Done!